docs/contribute/doc-release-process.md

```diff
@@ -0,0 +1,119 @@
+---
+description: Release process for documentation that uses a versioning system.
```

Comment:
```suggestion
description: Release a new version of the documentation.
```

Update description to start with verb, to match descriptions for other pages in this section.